### PR TITLE
Phantom JS and other small fixes

### PR DESCRIPTION
**Pull request by srirams6**

### CODE REVIEW

**Comment by sananth12** [comment]
Tested it?

**Comment by sananth12** [comment]
$ image-scraper --max-images 5 http://imgur.com/gallery/UbTOi
Traceback (most recent call last):
  File "/usr/local/bin/image-scraper", line 9, in <module>
    load_entry_point('ImageScraper==2.0.3', 'console_scripts', 'image-scraper')()
  File "/home/sananth12/Desktop/Github/srirams6/ImageScraper/image_scraper/__init__.py", line 38, in console_main
    download_path = os.path.join(os.getcwd(), save_dir)
  File "/usr/lib/python2.7/posixpath.py", line 75, in join
    if b.startswith('/'):
AttributeError: 'NoneType' object has no attribute 'startswith'

File: image_scraper/__init__.py [hunk 1]
```diff
@@ -24,24 +24,45 @@ def console_main():
...
-    except requests.exceptions.MissingSchema:
-        URL = "http://" + URL #Default schema is HTTP unless specified
-        page = requests.get(URL)
 
-    tree = html.fromstring(page.text)
+    no_to_download = args.max_images
 
-    img = tree.xpath('//img/@src')
+    save_dir = args.save_dir
+    if save_dir == 1: #argument not given
```
Comment (sananth12) [inline image_scraper/__init__.py]: replace with
if not save_dir :

caused the posted error

File: image_scraper/__init__.py [hunk 2]
```diff
@@ -59,8 +80,8 @@ def console_main():
     if not save_dir: #argument not given
         save_dir = "images"
     download_path = os.path.join(os.getcwd(), save_dir)
```
Comment (sananth12) [inline image_scraper/__init__.py]: download_path already set at line: 35

**Comment by srirams6** [comment]
Fixed the bug by setting the 'default' parameter for save_dir as 'images'.

**Pull request merged**